hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/DistributedObjectCounterCollector.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hazelcast.internal.util.phonehome;
+
+import com.hazelcast.collection.impl.list.ListService;
+import com.hazelcast.collection.impl.queue.QueueService;
+import com.hazelcast.collection.impl.set.SetService;
+import com.hazelcast.core.DistributedObject;
+import com.hazelcast.instance.impl.Node;
+import com.hazelcast.map.impl.MapService;
+import com.hazelcast.multimap.impl.MultiMapService;
+import com.hazelcast.ringbuffer.impl.RingbufferService;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.groupingBy;
+
+class DistributedObjectCounterCollector implements MetricsCollector {
+
+    @Override
+    public Map<String, String> computeMetrics(Node hazelcastNode) {
+
+        Collection<DistributedObject> distributedObjects = hazelcastNode.hazelcastInstance.getDistributedObjects();
+        List<String> list = Arrays.asList(MapService.SERVICE_NAME, SetService.SERVICE_NAME, QueueService.SERVICE_NAME,
+                MultiMapService.SERVICE_NAME, ListService.SERVICE_NAME, RingbufferService.SERVICE_NAME);
```

Comment:
Is it necessary to create this list every time when the `computeMetrics()` method is invoked? I think it could be a constant. Also, these service names are referred to one-by-one both here and line 52 - 57, which is a sort of a duplication (not any severe though). So I think it would be possible to have a service name -> metric name map (as a constant), and use it in the groupingBy() operation's classifier function to directly build a map which is keyed by metric name.  

Finally, please try to find a more meaningful name for this value than "list".